File: android/src/main/java/com/crype/plugins/capacitorelectronscreencapturer/ScreencapturerPlugin.java
package com.crype.plugins.capacitorelectronscreencapturer;

import com.getcapacitor.JSObject;
import com.getcapacitor.Plugin;
import com.getcapacitor.PluginCall;
import com.getcapacitor.PluginMethod;
import com.getcapacitor.annotation.CapacitorPlugin;

@CapacitorPlugin(name = "Screencapturer")
public class ScreencapturerPlugin extends Plugin {

    private Screencapturer implementation = new Screencapturer();

    @PluginMethod
    public void getSources(PluginCall call) {
        JSObject response = new JSObject();

        response.put("platform", "android");

        call.resolve(response);
    }
}

File: ios/Plugin/ScreencapturerPlugin.swift
import Foundation
import Capacitor

/**
 * Please read the Capacitor iOS Plugin Development Guide
 * here: https://capacitorjs.com/docs/plugins/ios
 */
@objc(ScreencapturerPlugin)
public class ScreencapturerPlugin: CAPPlugin {
    private let implementation = Screencapturer()

    @objc func getSources(_ call: CAPPluginCall) {
        call.resolve([
            "platform": "ios"
        ])
    }
}
